tests/test_todo_completion.py

```
@@ -0,0 +1,72 @@
+
+    def test_complete_todo_accepts_201_with_body(self):
+        """A 201 with a JSON body still returns the parsed payload."""
+        client = _client()
+        with patch.object(client, 'post', return_value=_response(201, '{"id": 99}')):
+            result = client.complete_todo('1', '2')
+        self.assertEqual(result.get('id'), 99)
+
+    def test_complete_todo_raises_on_error_status(self):
+        client = _client()
+        with patch.object(client, 'post', return_value=_response(403, 'Forbidden')):
+            with self.assertRaises(Exception):
```

Ruff (0.15.17)

[warning] Do not assert blind exception: `Exception`

(B017)